Makefile

```diff
@@ -381,7 +381,7 @@ binaries-from-container:
 	c_id=$$(docker create --name netplugin-build \
 		 netplugin-build:$(NETPLUGIN_CONTAINER_TAG)) && \
 	for f in netplugin netmaster netctl contivk8s netcontiv; do \
-		docker cp -a $${c_id}:/go/bin/$$f bin/$$f; done && \
+		docker cp $${c_id}:/go/bin/$$f bin/$$f; done && \
```

Comment:
this will affect the ownership, so tarballs/plugin/etc. might end up with a weird ownership, have you checked the outcome?  we may need to add in a chown in some places

Reply:
They were fine in the resulting tarball when I've checked. I think it's fine. Please feel free to check again.

Reply:
tested, binary tarball and v2plugin are both good